tests/data/ext/event_from_agent.xml

```diff
@@ -0,0 +1,26 @@
+<Data>
```

Comment:
Can you add an actual event from the agent instead of this one, which seems to have come from CustomScript? Heartbeat would be the simplest one, or ProcesSGoalState.

Reply:
Done.